Bottom: c9593a7ee50f48622d310bf30b98683ea51b4744
Top:    050591f89ab1109cc2ee5771fa46822a788676ba
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-02-17 20:55:34 +0000

Mark segments as not round before falling back to line-line intersection

This avoids assert failures with debugging enabeled


---

diff --git a/src/polygon1.c b/src/polygon1.c
index 58ca3fa..32d0ef7 100644
--- a/src/polygon1.c
+++ b/src/polygon1.c
@@ -1292,6 +1292,8 @@ seg_in_seg_arc_arc (struct info *i, struct seg *s1, struct seg *s2)
       s1->v->radius == s2->v->radius)
     {
       printf ("Cop-out for co-circular arcs for now - they will be better handled by their line approximations. <<<<<<<<<<\n");
+      s1->v->is_round = false;
+      s2->v->is_round = false;
       return seg_in_seg_line_line (i, s1, s2);
     }
 #endif
